pkg/challenges/challenges.go

```diff
@@ -61,21 +63,87 @@ const (
 )
 
 type ChallengeResult struct {
```

Comment:
can this now be made private?

Reply:
Yup! Want me to do that now? (This is slowly turning into the private implementation of an uber `Issuer` that we can break into seperate implementation)

Reply:
I'd like to avoid making this too much larger, it's hard to follow each of these changes and make sure nothing is breaking. If it's just making it private, I think that's fine, but if it touches a bunch of other files, let's hold off.

Reply:
Yeah sorry this one is big. I'll just make it private in a follow up